pom.xml

```diff
@@ -60,6 +60,7 @@
         <assertj.version>3.5.2</assertj.version>
         <logback.version>1.1.7</logback.version>
         <commonsIO.version>2.5</commonsIO.version>
+        <revision>1.1.43-SNAPSHOT</revision>
```

Comment:
Why an explicit number - it shows as if we're going to advance this number. Maybe something like 0-SNAPSHOT is better ?
Also, let's include a simple explanation:
```
<!--
   The version of this project is determined upon deployment using a a GitHub release name (i.e. tag). The version mentioned here is not a real version, and serves mainly for local builds
-->
```

Reply:
ok